### PR TITLE
sign in with redirect

### PR DESCRIPTION
**Pull request by cedvdb**

### CODE REVIEW

**Comment by vbourdeix** [comment]
@AnthonyNahas do you have an idea when this pull request could be merged and when will it be available in the node package ? For now I'm stuck as I couldn't successfully replace the official package by cedvdb's fork, and I need this signInWithRedirect possibility.

File: projects/ngx-auth-firebaseui/src/lib/interfaces/config.interface.ts [hunk 1]
```diff
@@ -29,10 +29,13 @@ export interface NgxAuthFirebaseUIConfig {
...
   // Control whether or not email verification is used
   enableEmailVerification?: boolean;
+
+  // whether we sign in with popup or redirect
+  authMethod: 'popup' | 'redirect';
 }
 
 export const defaultAuthFirebaseUIConfig: NgxAuthFirebaseUIConfig = {
-  // authMethod: 'redirect',
+  authMethod: 'popup',
```
Comment (AnthonyNahas) [inline projects/ngx-auth-firebaseui/src/lib/interfaces/config.interface.ts:38]: this should be customizable and not as the default value...

**Review by AnthonyNahas — COMMENTED** [review]
may you test please this PR with the latest state of the library... ?

there are some git conflicts...

**Comment by AnthonyNahas** [comment]
@vbourdeix Did you test that ?

**Comment by Blunderchips** [comment]
Is this a solution to #162 ?

**Comment by AnthonyNahas** [comment]
this should be optional 

This ticket should be updated and fixed to merge

**Pull request closed without merging**